src/applications/hca/sass/hca.scss

```diff
@@ -9,6 +9,19 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../platform/forms/sass/m-form-confirmation";
 
+.confirmation-page {
+  padding-bottom: 5rem;
+}
+
+.process-steps {
```

Comment:
Also, these rules might have an unintended impact on the existing "subway" map that lives on the IntroductionPage

Reply:
Both new "confirmation-page" & "process-steps" className exist only in ConfirmationPageNew.jsx -- verified via recursive folder-search -- so they should have no styling impact on orig. ConfirmationPage or HCASubwayMap.

If styling-impacts are the only concern, then this convo can be Resolved.  Leaving it unresolved in case the currently failing coverage test -- I'm suspecting it's because my updated ConfirmationPage.unit.spec.jsx unit-test is testing the orig page and not the gated, updated one.

Reply:
All tests have now passed, but leaving this convo to you to Resolve.

Reply:
Merge approved.